sputnikdao2/src/proposals.rs

```diff
@@ -91,6 +91,8 @@ pub enum ProposalKind {
     },
     /// Just a signaling vote, with no execution.
     Vote,
+    /// Update part of the policy.
+    UpdatePolicy { updates: Vec<PolicyUpdate> },
```

Comment:
From a naming point of view, `ChangePolicy` and `UpdatePolicy` types can lead to confusion for the end user, as they both seem to do the exact same thing.

Can we rephrase `UpdatePolicy` to avoid confusion? Some alternative names that pop into my mind now: `PartiallyUpdatePolicy` or `UpdatePolicyByJsonPaths` or `UpdatePolicyByRegex`.